indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -192,13 +192,19 @@
         data-centers (map #(data-center/data-center-short-name->elastic-doc kms-index %)
                           (map str/trim (data-center/extract-data-center-names collection)))
         ;; returns a list of {:start-date xxx :end-date yyy}
-        temporal-extents (->> temporal-extents
+        temporal-extents (->> collection-temporal-extents
                               ;; converts temporal-extents into a list of many
                               ;; {:BeginningDateTime xxx :EndingDateTime xxx}
                               (mapcat spec-time/temporal-ranges)
                               (map #(set/rename-keys % {:BeginningDateTime :start-date
                                                         :EndingDateTime :end-date}))
                               (map #(apply-function-to-all-values-in-map % index-util/date->elastic)))
+        temporal-extents-ranges (->> collection-temporal-extents
+                                     (map #(dissoc % :SingleDateTimes))
```

Comment:
Why do you need to exclude :SingleDateTimes?

Reply:
SingleDateTimes will always have an overlap of 0, so there is no reason to process them. In addition, when indexing a list elastic will remove dups so then it would be a single value, which would mess up the script.

Reply:
Got it. thanks.